.github/workflows/main.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+on:
+  - push
+  - pull_request
```

Comment:
No need to run CI on pushes to non-master branches
```suggestion
  push:
    branches: master
  pull_request:
```

Reply:
You’re right 👍🏻